### ISSUE TITLE
Youtube / twitch stutter

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#9044]
- Did you search for similar issues before submitting this one?

- Describe the issue you encountered:
Issue reported on community https://community.brave.com/t/youtube-twitch-stutter/3074
> I've tested this many times, while everything works fine in Chrome, it's stuttering once every minute on average when watching twitch of youtube videos, isn't html5 video player the same as in chrome? It's not a fixed timed stutter, it varies anywhere from 30sec up to 3-4minute but it's there, it's like a freeze for a second or two. While this is happening I'm not touch the mouse, so there is no cpu load beside the video itself. Also sometimes I feel like there's a small freeze while typing but it's rarer. 
>
>I have about 10 folders of bookmarks in total I'd say there are at least 4-500 pages bookmarked. I have no pinned tabs. I'm running usually 25-30 tabs at least, heavy sites, 1/3 exchanges, 1/3 forums, 1/3 youtube/twitch/video sites. The CPU is not overheating and is not spiking to 100% when these stutters happen, it is generally between 20-30% load when watching a video but since I couldn't fix this I moved back to chrome and everything is fine there. I would like to stay on brave but it's impossible with the stutter, after a few hours even typing stutters and page scrolling, it's like all actions are made in blocks and there's a delay between the next block. 

- Platform (Win7, 8, 10? macOS? Linux distro?):
Win 7 x64

- Brave Version (revision SHA):
Brave	0.15.2
rev	af7ef42
Muon	2.58.9
 

- Screenshot if needed:

- Any related issues:
cc: @bridiver @darkdh @bbondy

**Comment by admshao** [comment]
+1. Exactly same issue since 15.2 for sure. Can't remember on 15.1.

Running 15.7 on ArchLinux and still happens. Switched for Firefox for now. Youtube/Twitch is a must for me but what made me switch is the same lag on scrolls/inputs.

**Comment by bsclifton** [comment]
@alexwykoff didn't you open a similar issue (or issues)?

I removed Windows tag since this happens on multiple platforms

**Comment by LaurenWags** [comment]
+1 from community https://community.brave.com/t/youtube-videos-always-periodically-freeze-for-a-few-seconds/5669

**Comment by bsclifton** [comment]
I believe this is a duplicate of https://github.com/brave/browser-laptop/issues/10094 (which was fixed in 0.18.x).  Will mark issue as closed- please re-open if it's a different issue or if issue is still present in 0.18.22 or newer

**Comment by NumDeP** [comment]
Hi @bsclifton @luixxiul I just noticed this was closed when I assumed that this issue may have been moved to a later milestones due to current issue but fact of the matter is that this is still prevalent. Could you please have a look at the following links in the community and deduce whether they still need assessing. 

Perhaps you could temporarily added the investigation label again. Thanks.

- https://community.brave.com/t/videos-not-playing-properly-on-other-platforms/4785

- https://community.brave.com/t/youtube-embedded-player-does-not-work/409

**Comment by Shaflugi** [comment]
I've started seeing this issue very recently (within the past couple days). When watching YouTube videos, the playback will be fine for about 15 seconds, then freeze for about 5. During this time, the CPU spikes up from 6% to 25% (A full CPU core). This spike occurs even if the video is paused and in the background (my browser is freezing as I'm typing this :( )

I've disabled all extensions to see if that helps, but there's no difference, it still behaves the same way.
